CONTRIBUTION.md

```diff
@@ -64,3 +64,19 @@ Bug: T42
 The first line is the subject and should be no longer than 50 characters, the second line is always blank, and other
 lines should be wrapped at 80 characters. This allows the message to be easier to read on GitHub as well as in various
 git tools.
+
+### PR workflow
+
+- Before submitting a PR
+  - Format your code and fix all the spelling/grammatical mistakes.
+  - Limit the column length to 120 characters.
```

Comment:
Can you be explicit on `column length` of what? Code or comments within code or the commit message?

Reply:
Commit message format highlights the column length for a commit message, so will change this line to mention code as well as comments within code.